e2e/sample-apps/modular.js

```diff
@@ -122,7 +122,24 @@ async function callFunctions(app) {
   const callTest = httpsCallable(functions, 'callTest');
```

Comment:
`functions` isn't reassigned but it looks like `callTest` is, and does still need the `let`.

Reply:
lol. duh